docs/build.md

```diff
@@ -68,6 +69,25 @@ A `Build` resource can specify a Git source, together with other parameters like
 - `source.revision` - An specific revision to select from the source repository, this can be a commit or branch name.
 - `source.contextDir` - For repositories where the source code is not located at the root folder, you can specify this path here. Currently, only supported by `buildah`, `kaniko` and `buildpacks` build strategies.
 
+By default, the Build controller will validate that the Git repository exists. If the validation is not desired, users can define the `build.build.dev/verify.repository` annotation with `false`. For example:
+
+Example of a `Build` with the **build.build.dev/verify.repository** annotation, in order to disbale the `spec.source.url` validation.
+
+```yaml
+apiVersion: build.dev/v1alpha1
+kind: Build
+metadata:
+  name: buildah-golang-build
+  annotations:
+    build.build.dev/verify.repository: "false"
+spec:
+  source:
+    url: https://github.com/sbose78/taxi
+    revision: master
+```
+
+_Note_: The Build controller only validates two scenarios. The first one where the endpoint uses an `https` protocol, the second one when a `ssh` protocol (_e.g. `git@`_) is defined and none referenced secret was provided(_e.g. source.credentials.name_).
```

Comment:
technically speaking, users can still use `http` for accessing git repositories ... especially if they are private/local ones vs. actual github.com

I think we should allow for that as well

Reply:
We miss this one. I added it, please take a look and resolve this conversation if possible.